.changeset/honest-donuts-sparkle.md

```diff
@@ -0,0 +1,29 @@
+---
+'@apollo/server': minor
+---
+
+In the Apollo Server Landing Page Local config, you can now opt out of the telemetry that Apollo Studio runs in the
+embedded Sandbox & Explorer landing pages. This telemetry includes Google Analytics for event tracking and
+Sentry for error tracking.
+
+Example of the new config option:
+
+```
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  plugins: [
+    process.env.NODE_ENV === 'production'
+      ? ApolloServerPluginLandingPageProductionDefault({
+          graphRef: 'my-graph-id@my-graph-variant',
+          embed: {
+            runTelemetry: false
+          },
+        })
+      : ApolloServerPluginLandingPageLocalDefault({
+          embed: {
+            runTelemetry: false
+          },
+        }),
+  ],
+});
```

Comment:
Not sure how to suggest or correctly escape a ``` but this needs one to close.

Reply:
👍🏻 done